packages/gmsadumper/PKGBUILD

```diff
@@ -0,0 +1,31 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=gmsadumper
+_pkgname=gMSADumper
+pkgver=5.d9ca83f
+pkgrel=1
+pkgdesc='A tool that Reads any gMSA password blobs the user can access and parses the values.'
+arch=('any')
+groups=('blackarch' 'blackarch-misc')
+url='https://github.com/micahvandeusen/gMSADumper'
+license=('custom:unkown')
```

Comment:
https://github.com/micahvandeusen/gMSADumper/issues/3

Unlicensed repo are copyrighted and we can't use them.